linera-service/src/linera/main.rs

```diff
@@ -176,9 +176,11 @@ impl Runnable for Job {
                     } => block.timestamp,
                     _ => panic!("Unexpected certificate."),
                 };
-                context
-                    .update_wallet_for_new_chain(id, key_pair, timestamp)
-                    .await?;
+                if key_pair.is_some() {
+                    context
```

Comment:
Perhaps add a warning and suggest a command to add the chain later?

Reply:
We currently don't have such a command, I think. `assign` only works if you do have the key. We could add an `add-chain` command that works like `assign` but without a key? But then maybe `add-chain` should _replace_ `assign`, and the key should be an optional parameter?